lib/conjur_client.rb

```diff
@@ -30,7 +30,13 @@ def v5?
     end
 
     def version
-      (ENV['CONJUR_VERSION'] || 5).to_i
+      if !ENV['CONJUR_VERSION'] || ENV['CONJUR_VERSION'] == ""
```

Comment:
This should be done with a `case` statement probably

Reply:
TIL Ruby has case statements which.. makes sense